plugins/node/opentelemetry-plugin-pg/src/utils.ts

```diff
@@ -49,9 +49,18 @@ function getJDBCString(params: PgClientConnectionParams) {
   return `jdbc:postgresql://${host}:${port}/${database}`;
 }
 
+// Keep track of individual clients in the pool, increment each time a new client is used. If
+// a pool is not used then this will be different for every query that is run.
```

Comment:
why would someone need something like this ?

Reply:
For visibility into the connection pooling. It can helpful to see how many different connections were used over a time period and the percentage of requests each one served.

Reply:
so this should be rather converted to a metric (possible after converting pg-pool plugin to instrumentation). not the information that will be added into each span.